src/transformers/integrations.py

```diff
@@ -1005,13 +998,13 @@ def __init__(
         **neptune_run_kwargs
     ):
         if not is_neptune_available():
-            # TODO: NPT-12189 - Update the exception
             raise ValueError(
-                "NeptuneCallback requires neptune-client to be installed. Run `pip install neptune-client`."
+                "NeptuneCallback requires the Neptune client library to be installed. "
+                "To install the library, run `pip install neptune-client`."
             )
 
-        from neptune.new.metadata_containers.run import Run
         from neptune.new.internal.utils import verify_type
+        from neptune.new.metadata_containers.run import Run
```

Comment:
@Raalsky why do we do internal imports here? We shouldn't do that...

Reply:
Indeed... The things are even worse as it's common pattern for a few integrations:
- https://github.com/neptune-ai/neptune-optuna/blob/65e9dc1e168d022f38c1cf946379f39fe47e578e/neptune_optuna/impl/__init__.py#L32
- https://github.com/neptune-ai/neptune-tensorflow-keras/blob/371c43d1a922df141a61f11d0a75b38f1ebb05ba/neptune_tensorflow_keras/impl/__init__.py#L41

Reply:
Let's fix this before it breaks in future :)

Reply:
Is this PR the right place to change it?

Reply:
I don't think so. I've made a JIRA ticked in this context so you can just leave it for me :wink:

Reply:
Right, so @Herudaio just needs to retract his CR. Come on, Marcin! Enough vacationing!